tox.ini

```diff
@@ -1,26 +1,18 @@
 [tox]
 envlist =
-    py27-django{18,19,110,111}-{postgresql,sqlite}
-    py35-django{18,19,110,111,20}-{postgresql,sqlite}
+    py35-django{111,20}-{postgresql,sqlite}
     py36-django{20}-{postgresql,sqlite}
-    py27-django111-sqlite
 
 [testenv]
 setenv=
   PYTHONPATH = {toxinidir}
 basepython =
-    py27: python2.7
     py35: python3.5
     py36: python3.6
 deps =
-    py26: Pillow>=2.3.1,<4.0.0
-    py{27,34,35}: Pillow>=2.3.1
-    mock==1.0.1
+    py{34,35}: Pillow>=2.3.1
```

Comment:
Did you mean 35,36 here?

Reply:
Actually, I ran tox without this dependency (which is not in `requirements.txt` or `dev_requeriments.txt`) and the tests passed. @berinhard do we need it for some reason?